packages/context/src/app/AppProvider.tsx

```diff
@@ -20,6 +20,7 @@ export type App = {
 export type Page = {
     id: string;
     name: string;
+    slug: string;
```

Comment:
Thought I made this optional, but I guess I never saved the update. Will update.